deployment/ansible/deploy.yml

```diff
@@ -39,7 +39,7 @@
     project_root: "{{ ansible_env.HOME }}/node"
     genesis_accounts_path: "{{ project_root }}/data/aecore/.genesis/accounts.json"
     is_remote_package: false
-    public_ip: "{{ ansible_ssh_host|default(ansible_host) }}"
+    public_ip: "{{ ansible_ssh_host|default(ansible_host)|default(inventory_hostname) }}"
```

Comment:
It is not clear to me why this change needed. (I see commit message "fix public ip for static inventory" though unclear what issue was.)

Reply:
@lucafavatella `ansible_ssh_host` and  `ansible_host` are set by the dynamic inventories of OSS/AWS. If you use the static inventory alone (provided in this PR), we need `inventory_hostname`, although the automation does not use it, I often use it locally.